src/main/java/org/example/UserService.java

```diff
@@ -0,0 +1,108 @@
+package org.example;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.LinkedHashMap;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class UserService {
+
+    /**
+     * @param clients
+     * @param name
+     * @return
+     */
```

Comment:
For Javadoc I meant to add a description of what function does, kind of 
`Calculates the total age for a specific name.`